README.md

```diff
@@ -101,9 +93,9 @@ sudo apt install intel-basekit
 source /opt/intel/oneapi/compiler/latest/env/vars.sh
 ```
 
-You can check that the `icx` command can be found in the PATH with `which icx`.
+Ensure that the `icx` command can be found in the PATH with `which icx`.
```

Comment:
I would put the `which icx` in a block so that it catches the eye and can be copied.
Unfortunately I can't suggest via GitHub because it uses triple back quotes.

Reply:
I applied everything and pushed force this suggestion into the apply commit.